src/transformer/mockDefiner/mockDefiner.ts

```diff
@@ -137,6 +142,23 @@ export class MockDefiner {
         return this._declarationCache.get(declaration);
     }
 
+    public storeRegisterMockFor(declaration: ts.Declaration, factory: ts.FunctionExpression): void {
+        const key: string = this.getDeclarationKeyMap(declaration);
+
+        this._registerMockFactoryCache.set(declaration, key);
+        // this._factoryCache.set(declaration, key);
```

Comment:
I think you just forgot this line. If I understand the intention is good to have the registerFactoryCache separated so we can clean it when need it.

Reply:
removed, it was a leftover